app/views/layouts/application.html.slim

```diff
@@ -2,8 +2,8 @@ doctype 5
 html
   head
     title App Prototype
-    = stylesheet_link_tag    "application", media: "all"
-    = javascript_include_tag "application"
+    = stylesheet_link_tag              "application", media: "all", "data-turbolinks-track": "reload"
+    = javascript_packs_with_chunks_tag "application", 'data-turbolinks-track': 'reload'
```

Comment:
Just curious, how are the chunks configured? Or are they determined automatically somehow?

Reply:
Ah, I needed to add another line to enable it. https://github.com/rails/webpacker/blob/master/docs/webpack.md#add-splitchunks-webpack-v4

This seems pretty new and worth seeing how it works in the field.